linkml_model/model/schema/meta.yaml

```diff
@@ -512,6 +512,69 @@ slots:
     in_subset:
       - basic
 
+  enum_uri:
+    aliases:
+      - public ID
+    domain: enum_definition
+    range: uriorcurie
+    description: URI of the enum in an RDF environment
+    ifabsent: class_curie
+    in_subset:
+      - basic
+      
+  include:
+    domain: enum_expression
+    range: anonymous_enum_expression
+    multivalued: true
+
+  minus:
+    domain: enum_expression
+    range: anonymous_enum_expression
+    multivalued: true
+
+  inherits:
+    domain: enum_expression
+    range: enum_definition
+    multivalued: true
+
+  matches:
+    domain: enum_expression
+    range: match_query
+
+  identifier_pattern:
```

Comment:
`identifier_pattern`: Would like a definition for this field. Maybe some examples. I see range is `string` but I'm wondering if `regexp` is more appropriate? I'm assuming there is some syntax.